tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "isolatedModules": true,
+    "jsx": "react",
+    "lib": ["es6"],
```

Comment:
Is this really needed if you use `target` `esnext`?

Reply:
I have no clue. What would you suggest?

Reply:
I usually leave `target: "es6"` and it defaults to importing all the necessary libs.